lua/xcodebuild/tests/provider.lua

```diff
@@ -36,7 +36,7 @@ function M.find_tests(opts)
   local selectedTests = {}
 
   for _, line in ipairs(lines) do
-    selectedClass = string.match(line, "class ([^:%s]+)%s*:?")
+    selectedClass = string.match(line, "^%s*class ([^:%s]+)%s*:?")
```

Comment:
It will fail if there is some annotation like `@MainActor` or `final` keyword :(

Reply:
Maybe something like:
```
"^[\/]*class ([^:%s]+)%s*:?"
```

instead?

Reply:
@FelixLisczyk are you able to see if it works with that pattern?

Reply:
Yes, it works! I tested it in Neovim and with this online tool: https://reiynstudio.ink/lua-pattern-tester/  

Thanks! 👍

Reply:
> https://reiynstudio.ink/lua-pattern-tester/

I didn't know this website, thanks! I often struggle with lua patterns, it might be helpful.